SlingModelPersist/pom.xml

```diff
@@ -172,6 +172,17 @@
                 <artifactId>maven-bundle-plugin</artifactId>
                 <version>4.2.0</version>
                 <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Export-Package>org.apache.sling.models.injectors,org.apache.sling.models.persistor,org.apache.sling.models.persistor.annotations</Export-Package>
+                        <Private-Package>org.apache.sling.models.injectors.*,org.apache.sling.models.persist.*,org.apache.sling.models.persistor.*</Private-Package>
```

Comment:
@badvision - Why is setting the Private Packages explicitly here necessary? Shouldn't you be able to follow the standard packaging naming conventions?